index.d.ts

```diff
@@ -617,7 +617,16 @@ export type KillOptions = {
 	forceKillAfterTimeout?: number | false;
 };
 
+type StreamUnlessIgnored<
+	StreamIndex extends string,
+	OptionsType extends Options = Options,
+> = IgnoresStreamResult<StreamIndex, OptionsType> extends true ? null : Readable;
+
 export type ExecaChildPromise<OptionsType extends Options = Options> = {
+	stdout: StreamUnlessIgnored<'1', OptionsType>;
+
+	stderr: StreamUnlessIgnored<'2', OptionsType>;
```

Comment:
Leaving the documentation blank as we do not document this property.
Since the final type uses `& ChildProcess`, it turns out `childProcess.stdout|stderr` still shows the TypeScript documentation for it coming from Node.js' `ChildProcess`.